app/mailers/subscription_mailer.rb

```diff
@@ -23,4 +23,14 @@ def notify_note_liked(node, user)
     @footer = feature('email-footer')
     mail(to: node.author.email, subject: subject).deliver
   end
+
+  def notify_tag_added(node,tag)
+    given_tags = node.tags
+    users_to_email = tag.followers_who_dont_follow_tags(given_tags)
```

Comment:
Here you're calling it on an instance of DrupalNodeCommunityTag, not on its parent tag. But that model `belongs_to` Tag, so you can just say `tag.tag.followers_who_dont_follow_tags`... odd but should work.